lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -105,6 +105,13 @@ def self.filters_schema
                  has_default: true,
                  visibility: false
 
+          schema :timeline_zoom_level,
+                 type: 'String',
+                 required: false,
+                 writable: true,
+                 has_default: true,
+                 visibility: false
```

Comment:
The allowed values are not restricted here although the specification correctly specifies them to be.